### PR TITLE
Translate preprocessing/sequence.md into Japanese

### PR DESCRIPTION
**Pull request by henry0312**

### CODE REVIEW

File: sources/preprocessing/sequence.md [hunk 1]
```diff
@@ -57,10 +56,10 @@ Vector Space](http://arxiv.org/pdf/1301.3781v3.pdf)
...
 
-- __Return__: numpy array of shape `(size,)`.
+- __戻り値__: 形状が`(size,)`であるnumpy配列．
 
-- __Arguments__:
-    - __size__: size of the vocabulary considered.
-    - __sampling_factor__: lower values result in a longer probability decay (common words will be sampled less frequently). If set to 1, no subsampling will be performed (all sampling probabilities will be 1).
+- __引数__:
+    - __size__: 語彙数．
+    - __sampling_factor__: lower values result in a longer probability decay (common words will be sampled less frequently). 1が与えられたとき，サブサンプリングは行われません（全てのサンプリング確率が1になります）．
```
Comment (henry0312) [inline sources/preprocessing/sequence.md]: sorry, I don't understand what this mean..., but someone will translate, I guess 😆

Comment (fchollet) [inline sources/preprocessing/sequence.md]: It means: with lower values of `sampling_factor`, common (frequent) words will be sampled less often. Is that clear?

Comment (henry0312) [inline sources/preprocessing/sequence.md]: Thanks! I got it.

**Comment by henry0312** [comment]
updated

**Pull request merged**